modules/rpc/yarpc_options.go

```diff
@@ -71,3 +73,20 @@ func onewayInboundMiddlewareFromCreateInfo(mci service.ModuleCreateInfo) []middl
 	// Intentionally panic if programmer adds non-middleware slice to the data
 	return items.([]middleware.OnewayInbound)
 }
+
+func withGraph(graph dig.Graph) modules.Option {
```

Comment:
@glibsm  
Totally agree and we shouldn't use it as an object in `fx` as well. I believe this is a situation where global graph is required and should be restricted to be used only within DIG. That way we can have sane dig.Resolve, dig.Register APIs and restrict baggage passing use cases.